watcher/local/watcher.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package local
+
+import (
+	"perun.network/go-perun/channel"
+)
+
+type (
+	ch struct {
```

Comment:
Yeah, actually I named it `channel` at first, but it collided with the name of the package channel.

So used this name. Since this is an unexported variable and scope of usage is only two files.

I am still thinking if we could use an alternate, better name for this variable 🤔 ?

Reply:
Hm ok i dont have an alternative either. Lets leave it open maybe @matthiasgeihs has an idea.

Reply:
If it didn't affect the readability of the code much, I think `ch` is more descriptive than `regEntry` because, `regEntry` does not tell what is the entry all about.